tests/unit/api/main.py

```diff
@@ -512,7 +518,7 @@ def test_node_connect_network_success(self):
         # Check the actual HTTP response and status, not just the success;
         # we should do this at least once in the test suite, since this call
         # returns 202 instead of 200 like most things.
-        assert api.node_connect_network('node-99', '99-eth0', 'hammernet') == ('', 202)
+        assert api.node_connect_network('node-99', '99-eth0', 'hammernet') == ('', 202)  # noqa
```

Comment:
What line length are we limiting to? I measure this (pre `# noqa`) as 79 chars, which should be OK for pep8.